Source/VungleAdapterBannerAd.swift

```diff
@@ -41,24 +48,6 @@ final class VungleAdapterBannerAd: VungleAdapterAd, PartnerAd {
     func show(with viewController: UIViewController, completion: @escaping (Result<PartnerEventDetails, Error>) -> Void) {
         /// NO-OP
     }
-
-    /// Mapping of the request ad size to Vungle's ad size type. Nil means a MREC banner.
-    private var vungleAdSize: BannerSize {
-        switch request.size {
-        case IABStandardAdSize:
-            return BannerSize.regular
-        case CGSize(width: 300, height: 50):
-            return BannerSize.short
```

Comment:
I'm not sure why we originally supported this, since this case never would have happened.

I guess theoretically we could serve this size in an adaptive placement now, but since we don't support it otherwise, I feel like it makes sense to leave it out.